src/ui/ResultList/ResultList.ts

```diff
@@ -338,7 +338,7 @@ export class ResultList extends Component {
     Assert.exists(this.options.waitAnimationContainer);
 
     this.setupTemplatesVersusLayouts();
-    $$(this.root).on(ResultLayoutEvents.populateResultLayout, (e, args) => args.layouts.push(this.options.layout));
+    $$(this.root).on(ResultLayoutEvents.populateResultLayout, (e, args: IResultLayoutPopulateArgs) => args.layouts.push(this.options.layout));
```

Comment:
👍 It's a good idea to be as explicit as possible on all types being used. We're not doing that enough in general in the codebase (we rely on implicit return type a bit too much)